modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/RuntimeProvisioningService.java

```diff
@@ -23,45 +23,177 @@
 
 public interface RuntimeProvisioningService {
 
+    /**
+     * Set application details for the context
```

Comment:
I have created a JIRA for to fix this in allover the code.
https://wso2.org/jira/browse/APPFAC-3516